libgstc/libgstc.h

```diff
@@ -171,6 +171,18 @@ gstc_client_free (GstClient *client);
  */
 GstcStatus gstc_client_ping(GstClient *client);
 
+/**
+ * gstc_client_debug:
+ * @client: The client returned by gstc_client_new()
+ * @threshold: off (0), info (1), debug (2)
```

Comment:
Lets receive the string representing the threshold to allow for complex patterns